source/usage-examples/changeStream.txt

```diff
@@ -7,76 +7,80 @@ Watch for Changes
 Open a Change Stream
 --------------------
 
-You can keep track of changes to data in MongoDB, such as changes to a
-collection, database, or deployment, by opening a **change stream**. A change
-stream allows applications to watch for changes to data and react to them.
-You can open a change stream by calling the ``watch()`` method on
-a ``Collection``, ``Db``, or ``MongoClient`` object. The change stream
-returns **change event** documents when they occur.
+You can watch for changes to a single collection, a database, or an entire deployment in MongoDB with
+:manual:`Change Streams </changeStreams/>`.
```

Comment:
I intentionally moved any links to material outside of drivers/node from the top based on feedback we received about navigating away too quickly due to having the links featured too early. Would it be possible not to depend on the user visiting the manual link and feature the definition/summary of it here instead?